README.md

```diff
@@ -15,7 +15,7 @@ The App Center SDK for Unity supports the following platforms:
 * Android (5.0/API 21 or later)
 * UWP (Build 16299 or later)
 
-Also note that the App Center SDK for Unity is only available in C#.

```

Comment:
It's not directly related to this change, ~but it's definitely confusing nowadays.~ It was removed in 2018.3, so we still need to keep it because of support lower versions

JFYI - context about languages support :
- https://blogs.unity3d.com/2014/09/03/documentation-unity-scripting-languages-and-you/
- https://blogs.unity3d.com/2017/08/11/unityscripts-long-ride-off-into-the-sunset/